src/UI/Components/Diagram/Mock.ts

```diff
@@ -3820,7 +3827,8 @@ export const childModel: ServiceModel = {
   },
   config: {},
   service_identity: "name",
-
+  owner: "parent-service",
+  owned_entities: [],
```

Comment:
perhaps would be valuable to have a mock that has owned_entities and use it in a test case to cover that scenario?

Reply:
it's covered in the test third test case